go/client/cmd_dump_keyfamily.go

```diff
@@ -48,21 +63,31 @@ func (v *CmdDumpKeyfamily) Run() (err error) {
 	if !currentStatus.LoggedIn {
 		return fmt.Errorf("Not logged in.")
 	}
-	myUID := currentStatus.User.Uid
+
+	var UID keybase1.UID
+	if v.user != "" {
+		res := v.G().Resolver.Resolve(v.user)
```

Comment:
Ok I'll make that change, I'll also need to do that for `keybase.LoadPublicKeys`